cnames_active.js

```diff
@@ -3431,6 +3431,7 @@ var cnames_active = {
   "wdd": "wangduanduan.github.io",
   "weather": "phanendraguptha.github.io/weather",
   "weaver": "maxkfranz.github.io/weaver", // noCF? (don´t add this in a new PR)
+  "web-js-tutorial": "skyler1325.github.io/web-js-tutorial"
```

Comment:
```suggestion
  "web-js-tutorial": "skyler1325.github.io/web-js-tutorial",
```
As the CI error notes, you're missing a comma at the end of your entry.